### ISSUE TITLE
Marauder Stance Cooldowns

### ISSUE THREAD

**Issue by Vigfuss** [WarEmu/WarBugs#7717]
It's possible to trigger a stance cooldown if you press a greyed out stance button.  For example, when I have gift of savagery and stances have no cooldowns, if I press my savagery stance it will put all my stances on cooldown.

Nerfedbuttons can be used as a workaround.

**Comment by Luth-RoR** [comment]
Can confirm that and it is the same with the SW.
They probably have the same handling on the server, though worked differently.
The maras stances are greyed out when active and he couldn't deactivate his stance accidently on the mythic server (had to use "gift of release").
The SW stances could be deactivated accidently by using them again, because they had no dedicated _release stance_; this was high likely due to some design failure that has never been fixed by mythic.

**Comment by NorKalli** [comment]
A pain in the ass, as this can really screw you in combat if you activate a stance you're already in, but because each stance has its own primary action bar, you can place the other two stances within an active one that way you'll never trigger the stance you're already in- no NB required.

**Comment by Vigfuss** [comment]
Agreed Norkalli, that's a better workaround.  But the game also lets you hot key the stances, so you don't need to have the stances on your bar at all, these are having the same issue.
